### PR TITLE
Bug/20 button height

### LINKED ISSUE #20: Incorrect button size for Mat-Toggle-Buttons
#### Describe the bug
The buttons from Angular Material appear as 48px tall

#### What is the expected behavior?
In Figma, these buttons are 36px tall

#### What are the steps to reproduce?
1. Open the showcase demo
2. Inspect the toggle buttons
3. Observe that they are 48px tall

#### Screenshots or links to minimum reproduction example
image
versus
image
		
#### Anything else to add?
This is reported by Jonathan and the IPM team.

This comes from this rule in the angular material themes:
image

### PR DESCRIPTION
**Pull request by emclaug2**
Fixes #20 

#### Changes proposed in this Pull Request:
- MatButtonToggleGroup is now 36px height.

### CODE REVIEW

File: _common.scss [hunk 1]
```diff
@@ -13,4 +13,8 @@
     .mat-chip .mat-chip-avatar {
         font-size: 0.625rem;
     }
+
+    .mat-button-toggle .mat-button-toggle-label-content {
+        line-height: 36px;
+    }
```
Comment (daileytj) [inline _common.scss:19]: Not sure what the final result should be here. The button itself is 36px now, but with the border it's 38px. @huayunh what's the expected style here? Should the entire container, including the border be 36 or is it good to go as is?
Screen Shot 2021-09-23 at 8 45 59 AM
Screen Shot 2021-09-23 at 8 45 48 AM
.

Comment (huayunh) [reply _common.scss:19]: 36px.

Theoretically, it should be equally tall as a regular button, outlined or filled.

**Comment by emclaug2** [comment]
Ok the patch isn't as pretty but it should be pixel perfect now.  Toggle Group, including the borders, is 36px tall.

**Review by daileytj — APPROVED** [review]

**Review by huayunh — APPROVED** [review]
height looks cool with me

**Pull request merged**